ci/setup_gcs_cache.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+gcp_service_account_cleanup() {
+  echo "Deleting service account key file..."
+  rm -rf /tmp/gcp_service_account.json
+}
+
+if [[ ! -z "${GCP_SERVICE_ACCOUNT_KEY}" && ! -z "${BAZEL_REMOTE_CACHE}" ]]; then
```

Comment:
Do you want to grab me on slack and I can setup the env variables for you so we can test this?

Reply:
I was able to add those env variables to CircleCI